### PR TITLE
Add RekorUrl and PublicKey to ECP

### PR DESCRIPTION
**Pull request by lcarva**
https://issues.redhat.com/browse/HACBS-767

### CODE REVIEW

File: api/v1alpha1/enterprisecontractpolicy_types.go [hunk 1]
```diff
@@ -31,17 +31,30 @@ type EnterpriseContractPolicySpec struct {
...
 type Authorization struct {
-	ChangeID   string `json:"changeId"`
+	// The commit sha
+	// +optional
+	ChangeID string `json:"changeId"`
+	// The repo url containing the component sources
+	// +optional
 	Repository string `json:"repository"`
+	// The email address of the person authorizing the release
+	// +optional
```
Comment (zregvart) [inline api/v1alpha1/enterprisecontractpolicy_types.go:57]: Is this optional? cc @joejstuart

**Review by zregvart — APPROVED** [review]
Works for me, cc @joejstuart

File: api/v1alpha1/enterprisecontractpolicy_types.go (hunk 1 above)
Comment (lcarva) [reply api/v1alpha1/enterprisecontractpolicy_types.go:57]: (It is optional today)

Comment (zregvart) [reply api/v1alpha1/enterprisecontractpolicy_types.go:57]: Let's run with this :+1:

Comment (joejstuart) [reply api/v1alpha1/enterprisecontractpolicy_types.go:57]: Looks good, thanks. They are optional.

**Review by joejstuart — APPROVED** [review]

**Pull request merged**